recipe/meta.yaml

```
@@ -1,14 +1,14 @@
 {% set name = "pysimplegui" %}
-{% set version = "4.6.0" %}
+{% set version = "4.15.2" %}
 {% set styledName = "PySimpleGUI" %}
 
 package:
   name: "{{ name|lower }}"
   version: "{{ version }}"
 
 source:
-  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz"
-  sha256: "c3edb437bd50da95b9d7d205778ec85a595b61cfa9a6aeb2779d3b3da9a725ea"
+  url: "https://files.pythonhosted.org/packages/b2/d0/c20c020af7d2e61ab61ba2b2c9299b5eda75ab5615498733f1a04d8996c1/PySimpleGUI-{{ version }}.tar.gz"
```

You should change this back to the predictable URL:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ styledName }}-{{ version }}.tar.gz
```